app/controllers/billings_controller.rb

```diff
@@ -0,0 +1,39 @@
+class BillingsController < ApplicationController
+  skip_before_action :require_login, only: [:new, :create]
+
+  def index
+  end
+
+  def show
+  end
+
+  def new
+    if current_user
```

Comment:
From @lacuchilla:
I'm not sure what the billings controller is used for.
The new method in the billings controller could be cleaned up a bit. There's a bit of repetitive code. I'm not sure what is happening in this method.
There's some irregular indentation and whitespace. The end at line 33 should be moved to line 40.